addons/udes_stock/models/stock_warehouse_orderpoint.py

```diff
@@ -67,3 +67,63 @@ def unlink(self):
         Product._compute_nbr_reordering_rules()
         return res
 
+    @api.model
+    def _create_or_update_replenishment_rules(self, product, location, qty):
```

Comment:
Nit: This should be a public method (no leading underscore) as it is called from a different class.